release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2780,3 +2780,30 @@ link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-31"]
+=== RHBA-2022:1605 - {product-title} 4.9.31 bug fix update
+
+Issued: 2022-05-02
+
+{product-title} release 4.9.31, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1605[RHBA-2022:1605] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1604[RHBA-2022:1604] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6955866[{product-title} 4.9.31 container image list]
+
+[id="ocp-kubernetes-1-22-8-updates"]
+==== Updates from Kubernetes 1.22.8
+
+This update contains changes from Kubernetes 1.22.6 up to 1.22.8. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1226[1.22.6], link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1227[1.22.7], and link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1228[1.22.8].
+
+[id="ocp-4-9-31-bug-fixes"]
+==== Bug fixes
+* Previously, a rewrite in the Application Programming Interface (API) prevented the creation of `DualStack` services without specifying the `ipFamilyPolicy`. This fix alerts users of the upcoming API change, ensuring awareness of the required `ipFamilyPolicy` specification beforehand. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050632#[*BZ#2050632*])
```

Comment:
You could change "Application Programming Interface" to lowercase "application programming interface". The only related reference I can find in the ISG is as follows:

> Do not use application program interface   noun
> Do not use. Use "application programming interface".

Reply:
From a review of https://bugzilla.redhat.com/show_bug.cgi?id=2050632, my understanding is that from kube-apiserver 1.23 / OpenShift 4.10,  users explicitly specify either `ipFamilyPolicy: PreferDualStack` or `ipFamilyPolicy: RequireDualStack` for DualStack services to be valid. If I understand correct, regarding the OpenShift 4.9 context for these Release Notes, the change introduced is that there is now a warning stating that the API change in 4.10 is upcoming.

You could update this bug fix description to state this explicitly. This will prevent any ambiguity whereby readers might think that the API change applies to 4.9. For example:

> "In kube-apiserver 1.23 and {product-title} 4.10, users must specify either `ipFamilyPolicy: PreferDualStack` or `ipFamilyPolicy: RequireDualStack` for dual-stack services to be valid. Previously, no warning was provided in {product-title} 4.9 about the API requirement for {product-title} 4.10. This fix alerts users of the upcoming API change, ensuring awareness of the required `ipFamilyPolicy` specification beforehand."